### PR TITLE
release(v0.8.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by andrewrynhard**
This is the official v0.8.0-alpha.0 release.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: website/content/docs/v0.7/Reference/cli.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ talosctl cluster create [flags]
       --arch string                             cluster architecture (default "amd64")
       --cidr string                             CIDR of the cluster network (default "10.5.0.0/24")
       --cni-bin-path strings                    search path for CNI binaries (VM only) (default [/home/user/.talos/cni/bin])
-      --cni-bundle-url string                   URL to download CNI bundle from (VM only) (default "https://github.com/talos-systems/talos/releases/download/v0.7.0-alpha.8/talosctl-cni-bundle-${ARCH}.tar.gz")
+      --cni-bundle-url string                   URL to download CNI bundle from (VM only) (default "https://github.com/talos-systems/talos/releases/download/v0.8.0-alpha.0/talosctl-cni-bundle-${ARCH}.tar.gz")
```
Comment (smira) [inline website/content/docs/v0.7/Reference/cli.md:77]: why `-dirty`? looks like another bug probably, I think my regex is not correct which trims the version in talosctl

Comment (andrewrynhard) [reply website/content/docs/v0.7/Reference/cli.md:77]: yep, I think so.

Comment (smira) [reply website/content/docs/v0.7/Reference/cli.md:77]:
```diff
diff --git a/cmd/talosctl/cmd/mgmt/cluster/create.go b/cmd/talosctl/cmd/mgmt/cluster/create.go
index bf93efd2..f0586d03 100644
--- a/cmd/talosctl/cmd/mgmt/cluster/create.go
+++ b/cmd/talosctl/cmd/mgmt/cluster/create.go
@@ -544,7 +544,7 @@ func getDisks() ([]*provision.Disk, error) {
 
 func trimVersion(version string) string {
        // remove anything extra after semantic version core, `v0.3.2-1-abcd` -> `v0.3.2`
-       return regexp.MustCompile(`(-\d+-g[0-9a-f]+(-dirty)?)$`).ReplaceAllString(version, "")
+       return regexp.MustCompile(`(-\d+(-g[0-9a-f]+)?(-dirty)?)$`).ReplaceAllString(version, "")
 }
 
 func init() {
```

**Review by smira — DISMISSED** [review]

**Pull request merged**